cts/cli/regression.access_render.exp

```diff
@@ -52,30 +52,33 @@ A new shadow instance was created.  To begin using it paste the following into y
 * Passed: crm_attribute  - Enable ACLs
 =#=#=#= Begin test: An instance of ACLs render (into color) =#=#=#=
 The supplied command can provide skewed result since it is run under user that also gets guarded per ACLs on their own right. Continuing since --force flag was provided.
-<!-- ACLs as evaluated for user tony -->
+<!-- tony -->
 \x1b[34m<cib epoch="2" num_updates="0" admin_epoch="0">
-  <configuration>
-    <crm_config>
-      <cluster_property_set id="cib-bootstrap-options">
-        <nvpair id="cib-bootstrap-options-enable-acl" name="enable-acl" value="true"/>
-      </cluster_property_set>
-    </crm_config>
-    <nodes/>
-    <resources/>
-    <constraints/>
+  \x1b[34m<configuration>
+    \x1b[34m<crm_config>
+      \x1b[34m<cluster_property_set id="cib-bootstrap-options">
+        \x1b[34m<nvpair id="cib-bootstrap-options-enable-acl" name="enable-acl" value="true"/>[0m
+      \x1b[34m</cluster_property_set>[0m
+    \x1b[34m</crm_config>[0m
+    \x1b[34m<nodes/>[0m
+    \x1b[34m<resources/>[0m
+    \x1b[34m<constraints/>[0m
     \x1b[31m<acls>
-      <acl_role id="role-deny-acls">
-        <acl_permission id="deny-acls" kind="deny" xpath="/cib/configuration/acls"/>
-        <acl_permission id="read-rest" kind="read" xpath="/cib"/>
-      </acl_role>
-      <acl_target id="tony">
-        <role id="role-deny-acls"/>
-      </acl_target>
-    \x1b[31m</acls>
-  </configuration>
-  <status/>
-\x1b[34m</cib>[0m
-=#=#=#= Current cib after: An instance of ACLs render (into color) =#=#=#=
+      \x1b[31m<acl_role id="role-deny-acls">
+        \x1b[31m<acl_permission id="deny-acls" kind="deny" xpath="/cib/configuration/acls"/>[0m
+        \x1b[31m<acl_permission id="read-rest" kind="read" xpath="/cib"/>[0m
+      \x1b[31m</acl_role>[0m
+      \x1b[31m<acl_target id="tony">
+        \x1b[31m<role id="role-deny-acls"/>[0m
+      \x1b[31m</acl_target>[0m
+    \x1b[31m</acls>[0m
+  \x1b[34m</configuration>[0m
+  \x1b[34m<status/>[0m
+\x1b[34m</cib>[0m[0m
```

Comment:
I just noticed that only some lines end with the reset code, and the last line ends with it twice ... I don't think that causes any problems, but it points to something being odd

Reply:
I kept the extra one at the bottom of the XSL stylesheet for good measure. Upon further thought, I agree it is not necessary. Let me know if you'd like me to take it out.

Reply:
If we can guarantee that each line will have leading and ending codes, then we should take the final one out, otherwise leaving it in is better